spec/models/user_spec.rb

```diff
@@ -15,4 +18,20 @@
     expect(user).to_not be_valid
     expect(user.errors[:password_confirmation]).to_not be_blank
   end
+
+  describe '.admin?' do
```

Comment:
Yah, they were tests we missed when we finished up that branch. It was on the to-do list so I tacked it onto this commit. What would be a more appropriate way of handling this?